content/installing_nokogiri.md

```diff
@@ -176,6 +176,28 @@ ways.
         When working with this, be certain to use `$(brew --prefix libxml2)`
         because it will use the correct location for your Homebrew install.
 
+#### Unable to find libraries on macOS Mojave
+
+Xcode 10 on macOS Mojave [moves the system headers][xcode10] out of `/usr/include`
+and so Nokogiri will fail to build. Instead you'll see an error similar to this:
+
+```sh
+Building nokogiri using packaged libraries.
+-----
+libiconv is missing.  please visit http://nokogiri.org/tutorials/installing_nokogiri.html for help with installing dependencies.
+-----
+*** extconf.rb failed ***
+```
```

Comment:
I had a similar but not identical issue.  It never said `libiconv is missing`. Might be helpful to include the next line of the error as well.

```
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers. 
```

Reply:
Good idea - I've done that.